docs/sources/visualizations/geomap.md

```diff
@@ -12,6 +12,104 @@ The Geomap panel visualization allows you to view and customize the world map us
 
 {{< figure src="/static/img/docs/geomap-panel/geomap-example-8-1-0.png" max-width="1200px" caption="Geomap panel" >}}
 
+## Map View
+
+The map view controls the initial view of the map when the dashboard loads. 
+
+### Initial View
+
+The initial view configures how the GeoMap panel renders when the panel is first loaded.
+
+* **View** sets the center for the map when the panel first loads.
+* **Latitude** (available when the **View** mode is *Coordinates*)
+* **Longitude** (available when the **View** mode is *Coordinates*)
+* **Zoom** sets the initial zoom level for the GeoMap panel.
+
+## Data layer
+
+The Geomap visualization supports multiple Data Layers. Each data layer determines how you visualize geospatial data on top of the base map.
+
+### Layer Types
+
+There are four layer types to choose from in the Geomap visualization.
+
+* **Marker** renders a marker at each data point.
+* **Heatmap** visualizes a heatmap of the data.
+* **GeoJSON** renders static data from a geojson file.
+* **Icon at last point (alpha)** renders an icon at the last point in a series of points.
```

Comment:
Do we typically reference alpha features in panel docs (given general user won't see option unless they have opted in to alpha features)?

Reply:
You're right, I'll remove.